src/sentry/conf/server.py

```diff
@@ -746,6 +746,7 @@ def create_partitioned_queues(name):
     'organizations:integrations-v3': False,
     'organizations:invite-members': True,
     'organizations:new-settings': False,
+    'organizations:require-2fa': False,
```

Comment:
Do we have a release plan for this feature? i.e. is it for internal, early adopters, or everyone?

Reason why I'm asking is because you'll need to do some work on `getsentry` (better start prepping that env) :)

Reply:
@billyvg the feature is just for internal users until it's completely implemented. I believe @mattrobenolt didn't want this particular feature to be one large pull request, so I'm making slightly smaller ones, but to keep from having UI code that isn't functional, I'm putting it behind a flag for internal users only.